### PR TITLE
Whiskey2

### PR DESCRIPTION
**Pull request by WhiskeyKnight**
Ardatian

### CODE REVIEW

File: common/cultures/rathnir_cultures.txt [hunk 1]
```diff
@@ -366,4 +390,39 @@ luthic_group = {
...
 	}
 }
-	
\ No newline at end of file
+
+arcadian_group = {
+	graphical_culture = polynesiangfx
+
+	astanan = {
+		primary = A43 #Astana
```
Comment (Lokki-de-Nuuk) [inline common/cultures/rathnir_cultures.txt:398]: Astana is the capital city of Kazakhstan, we retconned it to something else, can't find it rn

File: common/cultures/rathnir_cultures.txt [hunk 2]
```diff
@@ -281,6 +281,26 @@ vastallosi_group = {
 	}
 }
 
+ardatian_group = {
```
Comment (Lokki-de-Nuuk) [inline common/cultures/rathnir_cultures.txt:284]: names are not alphabetised, you can use something like this: https://alphabetizer.flap.tv/

File: common/cultures/rathnir_cultures.txt [hunk 3]
```diff
@@ -366,4 +390,39 @@ luthic_group = {
...
+	kawalapuan = {
+		primary = A07 #Kawalapua
+
+		dynasty_names = {
+			Pantha Santha
+		}
+	}
+
+	dynasty_names = {
+		Valhorn "Twin-Leaf" Pantha Firnbolg Vishji
```
Comment (Lokki-de-Nuuk) [inline common/cultures/rathnir_cultures.txt:420]: repeated surnames; @Jabberdrake please clarify the EU4 syntax, will this cause issues?

File: common/cultures/rathnir_cultures.txt (hunk 1 above)
Comment (Lokki-de-Nuuk) [reply common/cultures/rathnir_cultures.txt:398]: Agriana; province ID: 387. Source: Mistic

File: common/cultures/rathnir_cultures.txt (hunk 3 above)
Comment (WhiskeyKnight) [reply common/cultures/rathnir_cultures.txt:420]: these should not cause issues, I dont know why, but they shouldn't

File: common/cultures/rathnir_cultures.txt (hunk 2 above)
Comment (WhiskeyKnight) [reply common/cultures/rathnir_cultures.txt:284]: names do not need to be alphabetized for code

File: common/cultures/rathnir_cultures.txt (hunk 1 above)
Comment (WhiskeyKnight) [reply common/cultures/rathnir_cultures.txt:398]: can strike these in main, the cultures are now defunct

**Pull request closed without merging**